README.md

```diff
@@ -25,4 +25,5 @@ Once your mod is complete and ready for use it can be compiled and reobfuscated
 ```
 
 ## License
-This template is available under the MIT license. Feel free to learn from it and incorporate it in your own projects.
+
+This template is available under the CC0 license. Feel free to learn from it and incorporate it in your own projects.
```

Comment:
Don't relicense the project. If you intended to, you forgot the LICENSE file anyway.

Reply:
The license file and fmj are already CC0, it's only the readme that says MIT